<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ÁΩëÁªúËÅäÂ§©Á®ãÂ∫è</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f8f9fa;
}

QTextEdit#chatDisplay {
    background-color: #ffffff;
    border: 1px solid #e9ecef;
    border-radius: 12px;
    padding: 15px;
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
    font-size: 14px;
    selection-background-color: #667eea;
}

QLineEdit#messageInput {
    border: 2px solid #e9ecef;
    border-radius: 20px;
    padding: 12px 20px;
    font-size: 14px;
    background-color: white;
    color: #333;
}

QLineEdit#messageInput:focus {
    border-color: #667eea;
    outline: none;
}

QPushButton#sendButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #667eea, stop:1 #764ba2);
    color: white;
    border: none;
    border-radius: 20px;
    padding: 12px 20px;
    font-size: 14px;
    font-weight: bold;
    min-width: 60px;
}

QPushButton#sendButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #5a67d8, stop:1 #6b46c1);
}

QPushButton#emojiButton {
    background-color: #f8f9fa;
    border: 2px solid #e9ecef;
    border-radius: 20px;
    padding: 8px;
    font-size: 16px;
    min-width: 40px;
    max-width: 40px;
}

QPushButton#emojiButton:hover {
    background-color: #e9ecef;
}

QGroupBox {
    font-weight: bold;
    border: 1px solid #dee2e6;
    border-radius: 8px;
    margin: 5px;
    padding-top: 15px;
    background-color: white;
    color: #495057;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 8px;
    color: #495057;
}

QComboBox {
    background-color: white;
    border: 1px solid #dee2e6;
    border-radius: 6px;
    padding: 8px 12px;
    font-size: 14px;
    color: #495057;
    min-width: 120px;
}

QLineEdit#hostInput {
    border: 1px solid #dee2e6;
    border-radius: 6px;
    padding: 8px 12px;
    font-size: 14px;
    background-color: white;
    color: #495057;
}

QPushButton#actionButton {
    background-color: #28a745;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton#actionButton:hover {
    background-color: #218838;
}

QLabel#connectionLabel {
    font-weight: bold;
    padding: 6px 12px;
    border-radius: 4px;
    background-color: #f8d7da;
    color: #721c24;
}

QStatusBar {
    background-color: #f8f9fa;
    color: #6c757d;
    border-top: 1px solid #dee2e6;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="connectionGroupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>ËøûÊé•ËÆæÁΩÆ</string>
      </property>
      <layout class="QHBoxLayout" name="connectionLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="modeLabel">
         <property name="text">
          <string>Ê®°Âºè:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="modeCombo">
         <item>
          <property name="text">
           <string>ÊúçÂä°Âô®Ê®°Âºè</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ÂÆ¢Êà∑Á´ØÊ®°Âºè</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hostLabel">
         <property name="text">
          <string>‰∏ªÊú∫:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hostInput">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
         <property name="placeholderText">
          <string>ËæìÂÖ•ÊúçÂä°Âô®IPÂú∞ÂùÄ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="actionButton">
         <property name="text">
          <string>ÂêØÂä®ÊúçÂä°Âô®</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="connectionLabel">
         <property name="text">
          <string>Êú™ËøûÊé•</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="chatDisplay">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="inputLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="emojiButton">
        <property name="text">
         <string>üòÄ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageInput">
        <property name="placeholderText">
         <string>ËæìÂÖ•Ê∂àÊÅØ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>ÂèëÈÄÅ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
